test/integration/utils_raft.py

```diff
@@ -0,0 +1,199 @@
+                self.uuid = _uuid
+                self.evts = evts  # keep reference, no copy
+                self.evt_tx_id = evt_tx_id
+
+            def cc_event(self, cc_event, block_number, tx_id, tx_status):
+                if tx_id in self.evts:
+                    if 'txEvents' not in self.evts[tx_id]:
+                        self.evts[tx_id]['txEvents'] = []
+                    self.evts[tx_id]['txEvents'] += [{
+                        'cc_event': cc_event,
+                        'tx_status': tx_status,
+                        'block_number': block_number,
+                    }]
+
+                # unregister chaincode event if same tx_id
+                # and disconnect as chaincode evt are unregister False
+                if tx_id == self.evt_tx_id:
+                    for x in self.evts[tx_id]['peer']:
+                        if x['uuid'] == self.uuid:
+                            x['channel_event_hub'].\
+                                unregisterChaincodeEvent(x['cr'])
+                            x['channel_event_hub'].disconnect()
+
+        o = CCEvent(_uuid, self.evts, self.evt_tx_id)
+        return o.cc_event
+
+    def registerChaincodeEvent(self, tx_id, cc_name, cc_pattern,
+                               channel_event_hub):
+        _uuid = uuid.uuid4().hex
+        self.evt_tx_id = tx_id
+        cr = channel_event_hub.registerChaincodeEvent(
+            cc_name, cc_pattern, onEvent=self.create_onCcEvent(_uuid))
+
+        if self.evt_tx_id not in self.evts:
+            self.evts[self.evt_tx_id] = {'peer': []}
+
+        self.evts[self.evt_tx_id]['peer'] += [
+            {
+                'uuid': _uuid,
+                'channel_event_hub': channel_event_hub,
+                'cr': cr
+            }
+        ]
+
+
+# This should be deprecated, and use client.get_user() API instead
+def get_peer_org_user(org, user, state_store):
+    """Loads the requested user for a given peer org
+        and returns a user object.
+    """
+
+    peer_user_base_path = os.path.join(
+        os.getcwd(),
+        'test/fixtures/e2e_cli/crypto-config/peerOrganizations/{0}'
+        '/users/{1}@{0}/msp/'.format(org, user)
+    )
+
+    key_path = os.path.join(
+        peer_user_base_path, 'keystore/',
+        E2E_CONFIG['test-network'][org]['users'][user]['private_key']
+    )
+
+    cert_path = os.path.join(
+        peer_user_base_path, 'signcerts/',
+        E2E_CONFIG['test-network'][org]['users'][user]['cert']
+    )
+
+    msp_id = E2E_CONFIG['test-network'][org]['mspid']
+
+    return create_user(user, org, state_store, msp_id, key_path, cert_path)
+
+
+def get_orderer_org_user(org='example.com', user='Admin', state_store=None):
+    """Loads the admin user for a given orderer org and
+        returns an user object.
+        Currently, orderer org only has Admin
+
+    """
+    msp_path = os.path.join(
+        os.getcwd(),
+        'test/fixtures/e2e_cli/crypto-config/ordererOrganizations/'
```

Comment:
same here

Reply:
This function is also not being used by the test, so I haven't changed it.